authentication/templates/change_password_email_msg.html

```diff
@@ -2,7 +2,7 @@
     <h2 style="margin: 0; padding: 10px; border-top-left-radius: 10px; border-top-right-radius: 10px; background-color: #39d69f86; color: #ffffff;">Verify Email</h2>
     <div style="margin: 0; padding: 15px; background-color: #ffffff;">
         <p>Hello,</p>
-        <p>Please copy the token below and enter it in the password recovery forms along with your new password.</p>
+        <p>Please copy the token below and enter it in the password recovery forms along with your new password. The token only valid for 30 seconds.</p>
```

Comment:
why only 30 seconds?

Reply:
Cause i think 3 days is too long. But i think i will change it to 5 minutes

Reply:
Nice, 5 mins is enough time but not too short